public/js/showoff.js

```diff
@@ -1274,16 +1268,5 @@ function setupStats()
 
 /* Is this a mobile device? */
 function mobile() {
-/*
-  return ( navigator.userAgent.match(/Android/i)
-            || navigator.userAgent.match(/webOS/i)
-            || navigator.userAgent.match(/iPhone/i)
-            || navigator.userAgent.match(/iPad/i)
-            || navigator.userAgent.match(/iPod/i)
-            || navigator.userAgent.match(/BlackBerry/i)
-            || navigator.userAgent.match(/Windows Phone/i)
-  );
-*/
-
   return ( $(window).width() <= 480 )
```

Comment:
I'm super not happy with this hack of a function. I was leaving the commented part there as a reminder of other ways of doing it. Do you have a suggestion for a better alternative?